tests/test_cli_get_set.py

```diff
@@ -195,6 +193,21 @@ def test_set_delete(self):
             self.assertNotIn('addresses:', out)
             self.assertNotIn('eth0:', out)
 
+    def test_set_delete_subtree(self):
+        with open(self.path, 'w') as f:
+            f.write('''network:\n  version: 2\n  renderer: NetworkManager
+  ethernets:
+    eth0: {addresses: [1.2.3.4/24]}''')
+        self._set(['network.ethernets=null'])
+        self.assertTrue(os.path.isfile(self.path))
+        with open(self.path, 'r') as f:
+            out = f.read()
+            # print(out, flush=True) # debugging
```

Comment:
This seems to be a copy & pasted left over from my debugging attempts a while ago. We can probably drop it, but it doesn't hurt in here either.

Reply:
Removed, and I credited you for the tests, as is proper :)

Reply:
Ha! I didn't even remember that dropped that test code on LP the other day xD